package.json

```diff
@@ -127,6 +127,7 @@
     "d3-zoom": "^3.0.0",
     "flux": "~4.0.1",
     "fs-extra": "^10.0.0",
+    "fsevents": "^2.3.3",
```

Comment:
Hi @Zubin-glitch we can't check in fsevents as a requirement because it only works on Mac computers. Please back out this change, and the package-lock.json file.